packages/core/src/internal-components/panel/panel.element.ts

```diff
@@ -0,0 +1,54 @@
+import { LitElement, html } from 'lit-element';
+import { baseStyles } from '@cds/core/internal';
+import { styles } from './panel.element.css.js';
+
+export const CdsInternalPanelTagName = 'cds-internal-panel';
+
+/**
+ *
+ * ```typescript
+ * import '@cds/core/internal-components/panel/register.js';
+ * ```
+ *
+ * ```html
+ * <cds-internal-panel>
+ *  // content
+ * </cds-internal-panel>
+ * ```
+ *
+ * @beta
+ * @element cds-internal-panel
+ * @slot - Content slot for the content inside the panel
+ * @cssprop --width
+ * @cssprop --height
+ * @cssprop --color
+ * @cssprop --overflow
+ * @cssprop --overflow-x
+ * @cssprop --overflow-y
+ * @cssprop --background
+ * @cssprop --border-color
+ * @cssprop --border-width
+ * @cssprop --border-radius
+ * @cssprop --padding
+ * @cssprop --box-shadow
+ */
+export class CdsInternalPanel extends LitElement {
+  connectedCallback() {
+    super.connectedCallback();
+    this.setAttribute('slot', 'panel');
```

Comment:
I don't think we want to add a default named slot to the panel. If this were to be projected into another component without a slot named `panel` or if needs to project multiple panels into different slots it will wont be able to render.  Components only get projected into another components default unnamed `<slot></slot>` if they don't have a slot name.